apps/website/package.json

```diff
@@ -14,7 +14,7 @@
     "@heroicons/react": "2.0.18",
     "@namehash/ens-utils": "1.3.0",
     "@namehash/ens-webfont": "workspace:*",
-    "@namehash/nameguard": "workspace:*",
+    "@namehash/nameguard": "0.1.3",
```

Comment:
@FrancoAguzzi is this intentional? The website should be able to install from the monorepo?